lib/jekyll_pages_api_search/search_hook.rb

```diff
@@ -12,13 +13,36 @@
 module Jekyll
   class Site
     alias_method :pages_api_after_render, :after_render
+    alias_method :orig_write, :write
+
+    def skip_index?
+      search_config = self.config['jekyll_pages_api_search']
+      search_config == nil || search_config['skip_index']
```

Comment:
That's what it's doing, though I've updated `search_config == nil` to `search_config.nil?`.